content/kubermatic/master/startio/_index.en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Start with KKP"
+weight = 1
+
++++
+
+![Kubermatic Kubernetes Platform logo](../img/KubermaticKubernetesPlatform-logo.jpg)
+
+## What is [start.kubermatic.io](https://start.kubermatic.io)?
+
+Application to simplify a way to bootstrap your Kubermatic Kubernetes Platform within a few minutes in an automated way operated with GitOps.
```

Comment:
A web-application that bootstraps the installation of your Kubermatic Kubernetes Platform and additional tools within a feew minutes in an automated way with applied GitOps principles.